src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -2,30 +2,31 @@
 
 # AudioInputProvider
 
-The `AudioInputProvider` provides a list of the user's available `audio-input` devices, the currently selected device.
+The `AudioInputProvider` provides a list of the user's available audio input devices and the currently selected audio input device.
 
 ## State
 
 ```javascript
```

Comment:
Since we have types in the code, can you make this typescript?

Reply:
Updated.